specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -7453,6 +7488,32 @@
           "type": "string",
           "readOnly": true
         },
+        "category": {
```

Comment:
Can you mark 'category' and 'acceptanceMode' as readonly? They're used only in the response of a GET request but this will keep the swagger clean.

Reply:
Sure! Done.